### DISCUSSION TITLE
Move skipped files to different folder?

### DISCUSSION THREAD

**Discussion by coreyaboy** [beetbox/beets#5349] [Q&A]
I am trying to configure beets to run in a particular way and I'm having a bit of trouble figuring it out on my own. Hopefully someone here can steer me in the right direction.

I want a command or script that will instruct beets to import a directory of newly acquired music without user input (-q /downloads) skipping files that it does not find a reasonable match with it's default behaviors, moving (-m) files that it does successfully import (I'm ok up to here) AND THEN I would like to move those skipped files over into a different "skipped items" folder to get them out of my music ingest folder.

In total I would have three relevant folders. A folder for my music library, a folder for music that is ready to be imported, and a folder for files that failed an unattended import. My intention is to come around and manually deal with the 'skipped' folder every now and then, while keeping the import folder clear of clutter.

I thought I could accomplish this by directing beets to write skips to a log file (-l /path/skipped.log), and then run a second import from that log (--from-logfile /path/skipped.log) where I instruct beets to import the files as-is, but move the files to my 'skipped' folder but I can't quite figure out how to give beets a different target path to move files to.

Can someone point me in the right direction here? There an easier way to do this?

**Comment by Serene-Arc** [comment]
Do you need beets to do this? It would be fairly easy to use a command like fd to move whatever is left in the original directory to another place where you can import as you desire? Unless you want it to be done incrementally?
